_pages/guides/oidc.md

```diff
@@ -0,0 +1,406 @@
+---
+title: OpenID Connect guide
+lead: >
+  <a href="http://openid.net">OpenID Connect</a> is a simple identity layer built on top of the OAuth 2.0 protocol. login.gov supports <a href="http://openid.net/specs/openid-connect-core-1_0.html">version 1.0</a> of the specification and conforms to the <a href="https://openid.net/wg/igov">iGov Profile</a>.
+redirect_from:
+  - /openid-connect/
+sidenav:
+  - text: Getting started
+    href: "#getting-started"
+  - text: Authorization
+    href: "#authorization"
+    links:
+      - text: Authorization response
+        href: "#authorization-response"
+  - text: Token
+    href: "#token"
+    links:
+      - text: Token response
+        href: "#token-response"
+  - text: User info
+    href: "#user-info"
+    links:
+      - text: User info response
+        href: "#user-info-response"
+  - text: Certificates
+    href: "#certificates"
+  - text: Logout
+    href: "#logout"
+    links:
+      - text: Logout response
+        href: "#logout-response"
+  - text: Example application
+    href: "#example-application"
+layout: page
+---
+
+# Getting started
+
+## Choosing an authentication method
+
+login.gov supports two ways of authenticating clients: **private_key_jwt** and **PKCE**.
```

Comment:
Is the goal to replace the developer docs site with this?

Reply:
To bring all partner-facing content together, yes. The goal is to have a single place for partners to get all the information they need to learn, decide, and implement login.gov.

Reply:
Ok! I noticed this page was already out of date from the developers site, what's the plan to unify these?

Reply:
Yea, I just copied content to see what it would look like here, but we'll need a real transition plan at some point.